### PR TITLE
Mago-MI - GDB/MI compatible frontend for Mago

### PR DESCRIPTION
**Pull request by buggins**
Changes in upstream code are minimal - just disabling some logging.

### CODE REVIEW

File: DebugEngine/MagoNatDE/MagoNatDE.vcxproj [hunk 1]
```diff
@@ -41,14 +41,14 @@
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
-    <ConfigurationType>DynamicLibrary</ConfigurationType>
```
Comment (rainers) [inline DebugEngine/MagoNatDE/MagoNatDE.vcxproj]: Changing the Debug/Release configurations should no longer be necessary. Just build with the "Debug/Release StaticDE" solution configuration. (It should still be possible to build MagoNatDE.dll as the debugger engine for VS).

File: ReadMe.txt [hunk 2]
```diff
@@ -1,3 +1,12 @@
+MagoMI fork:
+
+This is fork of Mago debugger which adds support of GDM MI compatible interface.
+
+See MagoMI/mago-mi/README.md for details.
+
+Original project: https://github.com/rainers/mago
```
Comment (rainers) [inline ReadMe.txt]: This note is no longer necessary in the main repo. The README might need some updates, though...

**Comment by rainers** [comment]
Apart from those 2 minor issues, looks good to me.

File: ReadMe.txt (hunk 2 above)
Comment (buggins) [inline ReadMe.txt]: fixed

File: DebugEngine/MagoNatDE/MagoNatDE.vcxproj (hunk 1 above)
Comment (buggins) [inline DebugEngine/MagoNatDE/MagoNatDE.vcxproj]: I hope now it's fixed

**Comment by rainers** [comment]
Thanks.

**Comment by rainers** [comment]
Now I can view the solution configurations, I'm a bit confused. The new projects are not enabled in the "StaticDE" configurations.
What versions do you actually build? Win32 and/or x64? I suspect you maintain your own solution. That's fine, but I'd like to be able to check for build errors, too.

BTW: I didn't see magoremote.exe in the dlangide build. This executable is necessary for debugging an x64 process by a 32-bit host. Maybe this is not a problem if the debugger itself is a 64-bit process. Is it all built into mago-mi.exe?

**Comment by rainers** [comment]
Another note: I noticed you have added the static mago libraries to the linker inputs of mago-mi.exe. Why don't you use project references for these? They provide better dependency handling...

**Comment by rainers** [comment]
I have cleaned it up for the Win32 builds locally. If it doesn't break builds for you, I can commit and push this.

**Comment by buggins** [comment]
feel free to fix them

**Comment by buggins** [comment]
I faced strange problem when trying to use fresh build of mago-mi.
Created issue #6.
Probably related to wrong versions of linked libraries.

**Comment by rainers** [comment]
> feel free to fix them

I pushed my changes.

**Comment by buggins** [comment]
wow. it fixed my issue

**Pull request merged**